backend-rust/src/graphql_api.rs

```diff
@@ -148,6 +151,8 @@ pub struct ApiServiceConfig {
     token_holder_addresses_collection_limit: u64,
     #[arg(long, env = "CCDSCAN_API_CONFIG_TOKEN_EVENTS_COLLECTION_LIMIT", default_value = "100")]
     token_events_collection_limit: u64,
+    #[arg(long, env = "CCDSCAN_API_CONFIG_NODE_STATUSES_CONNECTION_LIMIT", default_value = "100")]
+    node_statuses_collection_limit: u64,
```

Comment:
Since we have all of it in memory, this limit is not really protecting the service from anything, but it does not hurt either

Reply:
I will remove it. It was actually also the plan, since I have the Content-Length check